javascript/ql/src/LanguageFeatures/SpuriousArguments.ql

```diff
@@ -83,5 +83,13 @@ class SpuriousArguments extends Expr {
 
 from SpuriousArguments args, Function f, string arguments
 where f = args.getCall().getACallee() and
-      if args.getCount() = 1 then arguments = "argument" else arguments = "arguments"
+      if args.getCount() = 1 then arguments = "argument" else arguments = "arguments" and
+      (
+        // exclude empty functions, they are probably commented out debug utilities ...
+        exists(f.getABodyStmt()) or
+        // ... but include, constructors, arrows and externals
```

Comment:
Why are arrows special? And what about ambient TypeScript functions?

Reply:
> Why are arrows special? 

Arrows can not be empty, but they do not have to contain statements. 
Have you seen "practically empty" arrows in the wild that you want a heuristic for?